tags/browser/browser_mac.py

```diff
@@ -27,7 +27,7 @@ def address():
         except IndexError:
             return ""
         try:
-            web_area = window.element.children.find_one(AXRole="AXWebArea")
+            web_area = window.element.children.find(AXRole="AXWebArea")[-1]
```

Comment:
Is there a attribute of the correct webview that we can try to target instead (and add to the query), rather than trying to guess at the indices in the list? Seems like this would be more brittle over time

Reply:
Yes, we can filter the URL. I cleaned this up a bit so it filters dev tools web areas by URL on both Edge and Firefox; may need to do some more testing with further Chromium-based browsers as I don't have any others installed. I also replaced (slower) AppleScript with (faster) appscript for a backup strategy. Firefox has no useful scripting support so this would only work in Chromium-based browsers (and Safari, which has its own implementation).

https://github.com/nriley/talon_community/commit/4860223edead4f00a02d4470a08c3f0b733bc0e2